CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 ### Fixed
 * Label ListUsers API calls [#2000](https://github.com/openfga/openfga/pull/2000)
 * Fixed incorrect dispatch counts in List Objects [2013](https://github.com/openfga/openfga/pull/2013)
+* Fixed a bug in access control where write, create store, and list stores were not skippable [#2033](https://github.com/openfga/openfga/pull/2033)
```

Comment:
We don't need a changelog change for this PR considering the original feature hasn't been released yet